content/en/docs/overview/singleton-sdk/Angular Client Introduction/standalone-mode.md

```diff
@@ -21,52 +21,111 @@ The language normalization is required when the language comes from user browser
-| --directory |   string   | required |        The directory you want to put locale data in.         |
-|   --host    |   string   | required | This is singleton service which provides clients with translations and pattern. |
-|  --product  |   string   | required | Product name. For now, singleton service doesn’t explicitly restrict name of product, but it's better to keep short and sync with the name in release master. |
-| --component |   string   | required |                       Component name.                        |
-|  --version  |   string   | required |                       Release version.                       |
-|  --locales  |   string   | required | Specified the locale your product supports. Parameters are separated by commas, for example zh-CN,en-US. |
-|   --scope   |   string   | optional | Specified the i18n scope your product supports. Separated by , for example dates,numbers,currencies,dateFields. If not set, will not download the i18n formatting required data, only download the translation data. |
-|  --verbose  | Don't need | optional | If set, it will show all information during command execution for debug purpose. |
+```
+load-locale-data
+    -d ./src/assets/i18n/
+    --host https://g11n-vip-stg-1.eng.vmware.com:8090
+    --product Testing
+    --component AngularJS2
+    --version 1.0
+    --locales fr,es,ja,en-GB,pt-BR,de,ko,zh-TW,zh-CN
+    --scope dates,numbers,currencies,dateFields
 
-For example
+```
 
-![standalone-mode-2](https://github.com/zmengjiao/singleton/raw/website/content/en/images/standalone-mode/standalone-mode-2.png)
 
+##### **Add script in package.json**
 
-###### **Add script in package.json**
+```
+scripts: {
+    ...
+    "load-locale-data": "load-locale-data -d ./src/assets/i18n/ 
+                        --product NGXVIP --component demo 
+                        --host https://g11n-vip-stg-1.eng.vmware.com:8090 
+                        --version 1.0.0 
+                        --locales zh-Hans,en,ja,fr 
+                        --scope dates,numbers,currencies,dateFields"
+}
 
-![standalone-mode-3](https://github.com/zmengjiao/singleton/raw/website/content/en/images/standalone-mode/standalone-mode-3.png)
+```
 
-###### **Excute script at build time**
+##### **Execute script at build time**
 
-![standalone-mode-4](https://github.com/zmengjiao/singleton/raw/website/content/en/images/standalone-mode/standalone-mode-4.png)
+```
+npm run load-locale-data
+
+```
 
 
 ##### **Configure i18n assets**
 
 If use the local bundles instead of singleton service to support i18n, the asset path should be specified in singleton configuration, so in that case ngx-vip client will download the locale data from the path at runtime. Moreover, if the application also needs to handle the data formatting, please define the all functional categories which will be used in the configuration as i18nScope and make sure the downloading script contains the same relevant scope defintion.
```

Comment:
If use the local bundles instead of singleton service to support i18n, the asset path should be specified in singleton configuration, so in that case Singleton Angular client will download the locale data from the path at runtime. Moreover, if the application also needs to handle the data formatting, please define the all functional categories which will be used in the configuration as i18nScope and make sure the downloading script contains the same relevant scope definition.

Reply:
Done!